### PR TITLE
Update data preprocessing for Python 3

### PR DESCRIPTION
**Pull request by jli05**
* Update the `CODE_CENTRAL` URLs
* Update the data preprocessing code to Python 3

If you allow, I will update all the four sessions to Python 3.

### CODE REVIEW

**Comment by kyunghyuncho** [comment]
it looks like this PR changes the preprocessing code to work only with python3. Is it possible to maintain the backward compatibility with python2?

**Comment by jli05** [comment]
I made the scripts and `.py` files Python 2/3 compatible. Now we can invoke 

```bash
PYTHON=<python_interpreter> ./setup_local_env.sh
```

File: data/build_dictionary.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
+from __future__ import print_function
+
 import numpy
-import cPickle as pkl
+import pickle as pkl
```
Comment (orhanf) [inline data/build_dictionary.py]: @jli05 can we make it backward compatible with 

```python
try:
    import cPickle as pkl
except:
    import pickle as pkl
```

**Review by orhanf — CHANGES_REQUESTED** [review]

**Comment by jli05** [comment]
Done.

**Comment by orhanf** [comment]
Thanks for your efforts @jli05, merging

**Pull request merged**